ddtrace/contrib/trace_utils.py

```diff
@@ -60,6 +76,24 @@ def _normalized_header_name(header_name):
     return NORMALIZE_PATTERN.sub("_", normalize_header_name(header_name))
 
 
+def _get_header_value_case_insensitive(headers, keyname):
+    # type: (Mapping[str, str], str) -> Optional[str]
+    """
+    Get a header in a case insensitive way. This function is meant for frameworks
+    like Django < 2.2 that don't store the headers in a case insensitive mapping.
+    """
+    # just in case we are lucky
+    shortcut_value = headers.get(keyname)
+    if shortcut_value:
+        return shortcut_value
```

Comment:
nit: if `shortcut_value` is empty string it will not return but maybe it should

Reply:
True.